frontend/src/components/events/index.jsx

```diff
@@ -24,22 +25,42 @@ class Events extends Component {
       </Flex>
     );
 
+    const sponsorsHeading = (
```

Comment:
this should live in the Sponsors Component

Reply:
Thank you, Dan. I made the changes.